presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1190,6 +1190,12 @@ The table is partitioned by the transformed value of the column::
 
      ALTER TABLE iceberg.web.page_views ADD COLUMN ts timestamp WITH (partitioning = 'hour');
 
+ALTER VIEW
+
+Alter view operations to alter the name of an existing view to a new name is supported in the Iceberg connector::
```

Comment:
Should we clarify that we only supports altering views in Hive and Glue metastores? The same as `CREATE VIEW` above.